src/movies.js

```diff
@@ -2,14 +2,122 @@
 
 // Iteration 1: Ordering by year - Order by year, ascending (in growing order)
 
+function orderByYear(newArr) {
+  let array2 = [...newArr];
+  let orderByYear = array2.sort(function(objA, objB) {
+    if (objA.year < objB.year) {
+      return -1;
+    }
+    if (objA.year > objB.year) {
+      return 1;
+    }
+    if (objA.year === objB.year) {
+      if (objA.title < objB.title) {
+        return -1;
+      }
+      if (objA.title > objB.title) {
+        return 1;
+      }
+      if (objA.title === objB.title) {
+        return 0;
+      }
+    }
+  });
+  return orderByYear;
+}
+
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct
 
+/*Go ahead and create a howManyMovies() function that receives an array as a parameter and filter 👀 
+the array so we can have only the drama movies where Steven Spielberg is the director.
+*/
+
+function howManyMovies(array) {
+  /* if (num=0){
+
+    };
+    return num
+    */
+  if (array.length === 0) {
+    return 0;
+  }
+  const newArray = array.filter(function(obj, i, originalArr) {
```

Comment:
Just a little tip, try to avoid leaving unused parameters like i and originalArray.